<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>      PIC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>speed</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>229</y>
      <width>89</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>o</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>num1/num2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pos">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POSE</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>199</y>
      <width>75</width>
      <height>69</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>load index</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>load pose</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>load images</string>
     </property>
    </widget>
   </widget>
   <zorder>label_pic</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_pos</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>label_pos</zorder>
   <zorder>label_2</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_speed</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>255</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_load_index()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_load_pose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_load_images()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_oKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_xKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_slider_moved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_load_index()</slot>
  <slot>on_load_pose()</slot>
  <slot>on_load_images()</slot>
  <slot>on_oKey()</slot>
  <slot>on_xKey()</slot>
  <slot>on_text_changed()</slot>
  <slot>on_slider_moved()</slot>
 </slots>
</ui>
